src/Options.js

```diff
@@ -156,6 +157,7 @@ addFeatureOption('generatorComprehension', EXPERIMENTAL);
 addFeatureOption('memberVariables', EXPERIMENTAL);
 addFeatureOption('require', EXPERIMENTAL);
 addFeatureOption('strongMode', EXPERIMENTAL);
+addFeatureOption('strongModeRuntime', EXPERIMENTAL);
```

Comment:
Will this eventually be the flag associated with all of the dynamic aspects of strong mode? Some of these don't correspond so well to 'asserts', such as the various restrictions on strong mode functions (immutable bindings, lack of an 'arguments' object).